src/equations/compressible_euler_3d.jl

```diff
@@ -154,6 +154,7 @@ function initial_condition_weak_blast_wave(x, t, equations::CompressibleEulerEqu
 end
 
 
+# TODO: ICs, remove this one?
```

Comment:
What's the motivation for keeping this in 3D, if it is removed in 1D and 2D?

Reply:
We have elixirs for it in 1D and 2D but not in 3D, where we just have a dummy "set this as IC and let it run for a really short time although it doesn't make sense" test. I would vote to remove it but

Reply:
Given that it's trivial to reproduce if needed, I'd be ok to remove it.

Reply:
c695909c